apps/style/netsim/style.scss

```diff
@@ -268,35 +270,31 @@ svg {
       }
     }
 
-    div.panel_body {
+    div.panel-body {
       border-left: solid darkgray thin;
       border-right: solid darkgray thin;
       border-bottom: solid darkgray thin;
-      border-top: solid $code_org_pacific_blue thin;
+      border-top: solid $code-org-pacific-blue thin;
       border-radius: 0 0 $panel_border_radius $panel_border_radius;
     }
 
+    // Minimized panel, contents are hidden.
+    &.minimized {
+      h1 {
+        border-radius: $panel_border_radius;
+        .clear_button {
```

Comment:
Rule declaration should be preceded by an empty line<br>Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 4<br>Selector `clear_button` should be written in lowercase with hyphens